generators/Exercises/IsbnVerifier.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Generators.Input;
+
+namespace Generators.Exercises
+{
+    public class IsbnVerifier : Exercise
+    {
+        protected override void UpdateCanonicalData(CanonicalData canonicalData)
+        {
+            canonicalData.Exercise = "Isbn";
```

Comment:
In general, I would like to conform to the canonical data. Could you remove this line? This will cause the `Isbn` class to be renamed to `IsbnVerifier`, which still makes sense to me (and matches the file name).